components/brave_rewards/core/state/state_migration.h

```diff
@@ -44,19 +41,19 @@ class StateMigration {
 
   void OnMigration(ResultCallback callback, int version, mojom::Result result);
 
-  std::unique_ptr<StateMigrationV1> v1_;
-  std::unique_ptr<StateMigrationV2> v2_;
-  std::unique_ptr<StateMigrationV3> v3_;
-  std::unique_ptr<StateMigrationV4> v4_;
-  std::unique_ptr<StateMigrationV5> v5_;
-  std::unique_ptr<StateMigrationV6> v6_;
-  std::unique_ptr<StateMigrationV7> v7_;
-  std::unique_ptr<StateMigrationV8> v8_;
-  std::unique_ptr<StateMigrationV9> v9_;
-  std::unique_ptr<StateMigrationV10> v10_;
-  std::unique_ptr<StateMigrationV11> v11_;
-  std::unique_ptr<StateMigrationV12> v12_;
-  std::unique_ptr<StateMigrationV13> v13_;
+  StateMigrationV1 v1_;
```

Comment:
Nit: would you mind putting `const raw_ref<LedgerImpl> ledger_;` above `v1_` (also in `.cc` file)? I saw you did it elsewhere, so just for consistency.

Reply:
Done.